ex/loqui/lib/loqui/protocol/compressors/noop.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Loqui.Protocol.Compressors.NoOp do
+  @behaviour Loqui.Protocol.Compressor
+
+  def name,
+    do: " "
```

Comment:
It's an empty space because that's what the server sends. These names are the over-wire names, so naming it no-op would make it not work, since none of the servers call it no-op.